doc/man1/openssl-s_server.pod.in

```diff
@@ -781,6 +782,14 @@ instead of BIO_write() to send the HTTP response requested by a client.
 This option is only valid when B<-ktls> along with B<-WWW> or B<-HTTP>
 are specified.
 
+=item B<-zerocopy_sendfile>
+
+If this option is set, SSL_sendfile() will use the zerocopy TX mode, which gives
+a performance boost when used with kTLS hardware offload. Note that invalid
+TLS records might be transmitted if the file is changed while being sent.
+This option is only valid when B<-sendfile> is specified. Note that kTLS
```

Comment:
This statement isn't true, as doing `zerocopy_sendfile` turns on `sendfile` automatically (although with a warning). I would instead get rid of that warning, and just state that `zerocopy_sendfile` implies the `sendfile` options and leave it at that.

Reply:
It's the same as:

```
=item B<-sendfile>

If this option is set and KTLS is enabled, SSL_sendfile() will be used
instead of BIO_write() to send the HTTP response requested by a client.
This option is only valid when B<-ktls> along with B<-WWW> or B<-HTTP>
are specified.
```

I followed this existing example. If you think this pattern is not valid, I suggest fixing it outside the scope of this pull request.

Reply:
The problem I see is that you are saying an argument is _required_, yet it isn't. It's turned on automatically. In other words, the documentation says you have to specify:
```
openssl s_server -ktls -sendfile -zerocopy_sendfile
```
When in actuality, the following works.
```
openssl s_server -zerocopy_sendfiler
```
In this case, the `-sendfile` and `k'tls` options are enabled automatically and are _not_ required.

So, either the code needs to change to not automatically enable those options, or the documentation needs to change to state that `-ktls` and `-sendfile` are automatically enabled. Stating they may be required may be technically correct (the _best_ type of correct), but "required" has a specific meaning in terms of UI/UX.

Reply:
> So, either the code needs to change to not automatically enable those options, or the documentation needs to change to state that -ktls and -sendfile are automatically enabled.

I totally agree, and it makes total sense, but fixing `-sendfile` vs `-ktls` is outside of scope of my pull request, and fixing just `-zerocopy_sendfile` vs `-sendfile` will be inconsistent with `-sendfile` vs `-ktls`. Although, if you insist on writing the matching documentation for the new code, rather than consistency with the existing pattern, I can update the docs for `-zerocopy_sendfile`, and leave fixing the docs for `-sendfile` to someone else.

Reply:
At the very least, update the documentation to keep it correct with respect to the code. I actually don't think that it's too far out of scope to update the documentation for the `-sendfile`/`-ktls` option, since it's in the same file. But I would accept this PR without that change.

Reply:
It seems you understand where I'm coming from, so perhaps a later PR.